pkg/controller/controller.go

```diff
@@ -10,7 +12,7 @@ type Controller struct {
 }
 
 func Ping(c *gin.Context) {
```

Comment:
I deleted this func in main, so I think you will need to merge main and resolve the conflict